app/redux/weatherService.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios'
+import {createAsyncThunk, createSlice} from '@reduxjs/toolkit'
+
+export const fetchWeatherData = createAsyncThunk(
+  'weather/fetchWeatherData', async (city) => {
+  try {
+      const response = await axios.get(`https://api.openweathermap.org/data/2.5/forecast?q=${city}&appid=4d2230e28dd5ff342317c6687cd2d078`)
+      console.log(response)
+      return response.data
+    } catch (error) {
+      console.log('ur a bad coder')
```

Comment:
remove console logs as they are just used for troubleshooting - and not a bad coder, just need more practice :)